src/garage/experiment/deterministic.py

```diff
@@ -15,13 +15,17 @@ def set_seed(seed):
 
     """
     seed %= 4294967294
+    # pylint: disable=global-statement
     global seed_
     seed_ = seed
     random.seed(seed)
     np.random.seed(seed)
     if 'tensorflow' in sys.modules:
-        import tensorflow as tf
+        import tensorflow as tf  # pylint: disable= import-outside-toplevel
         tf.compat.v1.set_random_seed(seed)
+    if 'torch' in sys.modules:
+        import torch  # pylint: disable= import-outside-toplevel
```

Comment:
the [torch docs](https://pytorch.org/docs/stable/notes/randomness.html) mention a couple more steps necessary to make reproducibility work.

they also warn that using them can have a performance impact on GPU, so if we are going to set these options we should probably make it non-default and also throw up a warning banner about it to the console.

Reply:
I will create a separate PR for this if this is not a high priority.

Reply:
lgtm